Tests/MongoSwiftTests/ReadWriteConcernTests.swift

```diff
@@ -30,19 +30,19 @@ final class ReadWriteConcernTests: MongoSwiftTestCase {
     func testReadConcernType() throws {
         // check level var works as expected
         let rc = ReadConcern(.majority)
-        expect(rc.level).to(equal("majority"))
+        expect(rc.level).to(equal(ReadConcern.Level.majority))
```

Comment:
I think you can just say `.majority` rather than `ReadConcern.Level.majority`. Since `rc.level` has type `ReadConcern.Level`, the compiler should be able to infer the type.